### PR TITLE
add wildcard request body

### PR DESCRIPTION
**Pull request by anuraagnalluri**

### CODE REVIEW

**Comment by anuraagnalluri** [comment]
to support multiple endpoints for testing all nested cases of deserialization

File: cmd/server/main.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ func main() {
 		_, _ = w.Write([]byte("pong"))
 	}).Methods(http.MethodGet)
 	r.HandleFunc("/auth", auth.HandleAuth).Methods(http.MethodPost)
-	r.HandleFunc("/requestbody", requestbody.HandleRequestBody).Methods(http.MethodPost)
+	r.HandleFunc("/requestbody/{*}", requestbody.HandleRequestBody).Methods(http.MethodPost)
```
Comment (TristanSpeakEasy) [inline cmd/server/main.go:18]: don't really need this as in the openapi document you can use paths like `/requestBody#arrayOfArrays` to use the same path for different operations, you can see this in some of the auth paths I setup

**Review by TristanSpeakEasy — CHANGES_REQUESTED** [review]
I don't think this is needed

File: cmd/server/main.go (hunk 1 above)
Comment (anuraagnalluri) [reply cmd/server/main.go:18]: Ohh gotcha, thanks I'll do that. I did notice the `#` syntax you used earlier and did a quick search on the openapi docs for it, but didn't find anything at the time

**Pull request closed without merging**